CHANGELOG.md

```
@@ -93,6 +93,157 @@
+* removal of dependencies to geneinfo app ([#1635](https://github.com/varfish-org/varfish-server/issues/1635)) ([#1719](https://github.com/varfish-org/varfish-server/issues/1719)) ([48c1728](https://github.com/varfish-org/varfish-server/commit/48c17282b3fdce55cddccf0939655ebfe75104ab))
+* remove old filter features ([#739](https://github.com/varfish-org/varfish-server/issues/739)) ([#1628](https://github.com/varfish-org/varfish-server/issues/1628)) ([1ec5b3e](https://github.com/varfish-org/varfish-server/commit/1ec5b3e990372873282ba6ecd1bed576ab3ab243))
+* remove references to old hgnc gene table in varfish ([#1551](https://github.com/varfish-org/varfish-server/issues/1551)) ([#1633](https://github.com/varfish-org/varfish-server/issues/1633)) ([dbe4fd4](https://github.com/varfish-org/varfish-server/commit/dbe4fd4352eae8a96d69243c822ba2ceae9cb9ce))
+* searching for cases is painful ([#1645](https://github.com/varfish-org/varfish-server/issues/1645)) ([#1802](https://github.com/varfish-org/varfish-server/issues/1802)) ([5064a27](https://github.com/varfish-org/varfish-server/commit/5064a27ed8547e25d2529ad7e3c4b200fa4ceada))
+* separator for gene list or hpo terms ([#1196](https://github.com/varfish-org/varfish-server/issues/1196)) ([#1480](https://github.com/varfish-org/varfish-server/issues/1480)) ([ba73b99](https://github.com/varfish-org/varfish-server/commit/ba73b9959c2bd8acbf576c3a4e18ee926058d620))
+* seqvar item modified & revert  ([#1818](https://github.com/varfish-org/varfish-server/issues/1818)) ([c885421](https://github.com/varfish-org/varfish-server/commit/c8854214fe15c0c97fa6bc34230eb2f3272883eb))
+* seqvar panels ([#1883](https://github.com/varfish-org/varfish-server/issues/1883)) ([9d18422](https://github.com/varfish-org/varfish-server/commit/9d18422001108e7fc70bf65c5f5dcc9177e344bd))
+* seqvar quality, clinvar & locus filters ([#1856](https://github.com/varfish-org/varfish-server/issues/1856)) ([cf00085](https://github.com/varfish-org/varfish-server/commit/cf00085630a4e59c6481b349a1a5aaa1e45ae78b))
+* seqvars data-table ([#1870](https://github.com/varfish-org/varfish-server/issues/1870)) ([d5cb17b](https://github.com/varfish-org/varfish-server/commit/d5cb17bf7f6f87333c15c477d6ce99f9d43e0684))
+* seqvars effects filter ([#1833](https://github.com/varfish-org/varfish-server/issues/1833)) ([38febb1](https://github.com/varfish-org/varfish-server/commit/38febb1c55df3243054fbeb45b34ab2cc132e1eb))
+* seqvars summary item show modified + revert ([#1890](https://github.com/varfish-org/varfish-server/issues/1890)) ([5f1e1fd](https://github.com/varfish-org/varfish-server/commit/5f1e1fdbcefabcb08497524b097e899af1b5f6f5))
+* set default query preset for project ([#1478](https://github.com/varfish-org/varfish-server/issues/1478)) ([#1611](https://github.com/varfish-org/varfish-server/issues/1611)) ([a1ab0ff](https://github.com/varfish-org/varfish-server/commit/a1ab0ff55567f1292238e92301102677cc2bbaf1))
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)